doc/changelog/04-tactics/11029-add-strategy.rst

```diff
@@ -0,0 +1,3 @@
+- New tactic :tacn:`strategy` added which behaves like the vernacular
```

Comment:
Nobody is really sure what "vernacular" means, whereas everyone agrees on the definition of a "command", so better use the latter:

```suggestion
- New tactic :tacn:`strategy` added which behaves like the command
```